main/src/main/java/cgeo/geocaching/apps/cache/WhereYouGoApp.java

```diff
@@ -20,7 +20,8 @@
 import org.apache.commons.lang3.StringUtils;
 
 public class WhereYouGoApp extends AbstractGeneralApp {
-    private static final Pattern PATTERN_CARTRIDGE = Pattern.compile("(https?" + Pattern.quote("://www.wherigo.com/cartridge/") + "(details|download)" + Pattern.quote(".aspx?") + "[Cc][Gg][Uu][Ii][Dd]=[-0-9a-zA-Z]*)");
+    private static final Pattern PATTERN_CARTRIDGE = Pattern.compile("https?" + Pattern.quote("://www.wherigo.com/cartridge/") + "(details|download)" + Pattern.quote(".aspx?") + "[Cc][Gg][Uu][Ii][Dd]=([-0-9a-zA-Z]*)");
+    private static final String URL_BASE = "https://www.wherigo.com/cartridge/details.aspx?CGUID=";
```

Comment:
As we want use the download URL it would make sense to use the `/download.aspx` URL.

Reply:
Whereyougo handles both URLs the same way, so it doesn't make a difference.

Reply:
Sure, but it is more logical when reading the code.